src/Catty/Parser&Serialializer/ProjectMigrator.swift

```diff
@@ -22,7 +22,7 @@
 
 @objcMembers class ProjectMigrator: NSObject, ProjectMigratorProtocol {
 
-    static let minimumCatrobatLanguageVersionForScenes: Float = 0.992
+    static let minimumCatrobatLanguageVersionForScenes: Float = 0.993
```

Comment:
Projects must always be migrated to the new file structure starting from CBL 0.992. Hence, please do not change this value.